prow/scripts/cluster-integration/kyma-gke-integration.sh

```diff
@@ -207,6 +201,13 @@ yes | kyma install \
   --tls-key="${TLS_KEY}" \
   --timeout 60m
 
+log::info "Patch serverless to enable containerd"
+cp -va "${TEST_INFRA_SOURCES_DIR}/prow/scripts/resources/containerd-gke-patch.tpl.yaml" \
+       "${KYMA_SOURCES_DIR}/resources/serverless/templates/containerd-gke-patch.yaml"
+helm template -s templates/containerd-gke-patch.yaml resources/serverless/ --dry-run > "$PWD/patch-containerd.yaml"
+kubectl apply -f "$PWD/patch-containerd.yaml"
+sleep 1m
```

Comment:
We are waiting for the pods to be replaced. The manifest we are applying updates the Daemonset that injects the certificates on the nodes. Technically it's possible to `kubectl wait` or watch here, but it would be a bit complicated since the selectors are configurable. Simplest solution was to give it a minute.

Reply:
Shall we use fixed label and wait for all deamon sets to finish their jobs?
I am afraid of introducing race conditions, which are not easy to debug.

Reply:
Makes sense. I will add it :+1: